lib/src/components/NxSmallVulnerabilityCounter/NxSmallVulnerabilityCounter.scss

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use "sass:math";
+@use "../../scss-shared/nx-text-helpers";
+
+@mixin counter-colors {
+  @each $severity in critical, high, medium, low, none {
```

Comment:
I don't see designs for the "none" vuln ratings level in figma.

Reply:
> I don't see designs for the "none" vuln ratings level in figma.

We did confirm that "none" is part of this and the corresponding colors, but will let design know to have it in Figma.

Reply:
> > I don't see designs for the "none" vuln ratings level in figma.
> 
> We did confirmd that "none" is part of this and the corresponding colors, but will let design know to have it updated in Figma.

"None" have been added to the mockup:
https://www.figma.com/design/jG8RQhjPyIbisYAxDlOyt7/Vulnerability-Threat-Level-Count?node-id=0-1&node-type=canvas&t=K0fPH7oF4Mzyg3ZD-0